whois-api/src/test/java/net/ripe/db/whois/api/HazelcastInstanceTestManager.java

```diff
@@ -0,0 +1,56 @@
+package net.ripe.db.whois.api;
+
+
+import com.hazelcast.config.Config;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import net.ripe.db.whois.common.profiles.WhoisProfile;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+import java.util.List;
+
+@Configuration
+@Profile({WhoisProfile.TEST})
+public class HazelcastInstanceTestManager {
```

Comment:
Do no start this.
There is a task for testing this.
Just test the filter in this class